packages/shared/src/lib/dateFormat.ts

```diff
@@ -10,8 +10,59 @@ import {
 export const oneMinute = 60;
 export const oneHour = 3600;
 export const oneDay = 86400;
+const oneWeek = 7 * oneDay;
+const oneMonth = 28 * oneDay;
 export const oneYear = oneDay * 365;
 
+const publishTimeV1 = (
```

Comment:
Do we need `V1` on this? 😬

Reply:
Open to suggestions it's a experiment variation, so wasn't sure what to call it 😅

Reply:
Just thinking we don't need the V1 suffix so that we would need to go back and remove the V1 after the experiment, like with the layout. I'm indifferent either way